pkg/scd/operational_intents_handler.go

```diff
@@ -365,7 +365,7 @@ func (a *Server) UpdateOperationalIntentReference(ctx context.Context, req *rest
 
 type validOIRParams struct {
 	id             dssmodels.ID
-	ovn            restapi.EntityOVN
+	ovn            scdmodels.OVN
```

Comment:
Thanks, that's a convenient change

Reply:
How do we/will we differentiate between a client specifying an OVN as part of a 'normal' mutation (where the current OVN needs to be provided)?

(I see that #1078 mentions a new `requested_ovn_suffix` parameter which I assume will be added in upcoming PRs?)

Reply:
> (I see that https://github.com/interuss/dss/issues/1078 mentions a new requested_ovn_suffix parameter which I assume will be added in upcoming PRs?)

Exactly